.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: N6DBsOOlDU8w34t9iKM4jSIJTFqkhcS4j
```

Comment:
I know nothing about this, but is there opportunity/motivation for this to be a secret?